.devcontainer/import time.py

```
@@ -0,0 +1,92 @@
+import time
+import json
+from bitcoin import *
```

Ruff (0.8.2)

`from bitcoin import *` used; unable to detect undefined names

(F403)